src/utils/properties.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Gets an environment variable. If the env var is not set and a default value is not
+ * provided, then it is assumed it is a mandatory requirement and an error will be
+ * thrown.
+ */
+
+const getEnvironmentVariable = (key: string, defaultValue?: any): string => {
```

Comment:
These logic (improved) has already been implemented here https://github.com/companieshouse/overseas-entities-web/blob/main/src/config/index.ts

Reply:
Removed properties file and put this into config.